lib/forms/CoBraLab-Mouse-Touchscreen-Form/index.ts

```diff
@@ -0,0 +1,237 @@
+/* eslint-disable perfectionist/sort-objects */
+
+const { defineInstrument } = await import('/runtime/v1/@opendatacapture/runtime-core/index.js');
+const { z } = await import('/runtime/v1/zod@3.23.6/index.js');
+
+export default defineInstrument({
+  kind: 'FORM',
+  language: 'en',
+  tags: ['Mouse, Touchscreen','PVD','5-choice'],
+  internal: {
+    edition: 1,
+    name: 'MOUSE_TOUCHSCREEN_FORM'
+  },
+  content: {
+    experimentType: {
+      kind: 'string',
+      variant: 'radio',
+      label: "Type of touchscreen session",
+      options:{
+        "PVD":"PVD (Pairwise Discrimination)",
+        "5-choice": "5CSRTT (5 choice serial reaction time task)"
+      }
+    },
+    experimentStage: {
+      kind: 'string',
+      variant: 'select',
+      label: "Experiment stage",
+      options: {
+        "habituation":"Habituation",
+        "habituation2a":"Habituation2a",
+        "habituation2b":"Habituation2b",
+        "intial touch": "Initial touch",
+        "must touch": "Must touch",
+        "must initiate": "Must initiate",
+        "punish incorrect":"Punish incorrect",
+        "full session": "Full session"
+      }
+    },
+    pvdStage: {
+      kind: 'dynamic',
+      deps: ['experimentType', 'experimentStage'],
+      render(data){
+        if(data.experimentType === "PVD" && data.experimentStage === "full session"){
+          return {
+            kind: 'string',
+            variant: 'radio',
+            label: "PVD experiment stage",
+            options: {
+              "aquisition": "Aquisition",
+              "reversal":"Reversal",
+              "re-reversal": "Re-reversal"
+            }
+          }
+        }
+        return null
+      }
+    },
+    fiveChoiceStage: {
+      kind: 'dynamic',
+      deps: ['experimentType', 'experimentStage'],
+      render(data){
+        if(data.experimentType === "5-choice" && data.experimentStage === "full session"){
+          return {
+            kind: 'string',
+            variant: 'radio',
+            label: "5-choice experiment stage",
+            options: {
+              "4 second stimulus": "4 second stimulus",
+              "2 second stimulus": "2 second stimulus",
+              "Pro trial": "Pro trial"
+            }
+          }
+        }
+        return null
+      }
+    },
+    chamberNumber: {
+      kind: 'number',
+      variant: 'select',
+      label: 'Chamber number',
+      options: {
+        1: "one",
```

Comment:
I don't think we need to mix words and numbers here, its very weird.

Try another variant?

Should this just be a zod-restricted input field instead?

Reply:
Yeah the select and radio forcing you to have a number label is weird, I will switch it to limited number input instead